test/functional/p2p_invalid_messages.py

```diff
@@ -9,12 +9,14 @@
 from test_framework.messages import (
     msg_ping,
     ser_string,
+    MAX_PROTOCOL_MESSAGE_LENGTH
```

Comment:
19272: feels like we miss some previous backports... might want to investigate further/drop this one for now